code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -254,20 +254,33 @@ datum
 			penetrates_skin = 1
 			blob_damage = 5
 			value = 7 // 3 2 1 heat
+			var/counter = 1
 
 			on_mob_life(var/mob/M, var/mult = 1) // -cogwerks. previous version
 				if (!M) M = holder.my_atom
+				if (!counter) counter = 1
 				M.take_toxin_damage(1.5 * mult)
 				if (probmult(8))
 					M.emote("drool")
 				if (prob(10))
 					boutput(M, "<span class='alert'>You cannot breathe!</span>")
 					M.losebreath += (1 * mult)
 					M.emote("gasp")
-				if (prob(8))
-					boutput(M, "<span class='alert'>You feel horribly weak.</span>")
-					M.setStatus("stunned", max(M.getStatusDuration("stunned"), 30 * mult))
-					M.take_toxin_damage(2 * mult)
+				switch(counter+= (1 * mult))
+					if (20)
+						boutput(M, "<span class='alert'>You feel a bit weak.</span>")
+						M.setStatus("stunned", max(M.getStatusDuration("stunned"), 15 * mult))
```

Comment:
I haven't touched statuses before, possible these should be using the various time macros we have (`SECONDS` etc.)?

Reply:
These were like this before, i have just changed the numbers to make it not as strong early on. The method i used is the same as the original cyanide.

Reply:
> The method i used is the same as the original cyanide.

Nope. Previously there was an 8% chance of adding (assuming `mult` is 1) 3 seconds of stun. Now, for every tick between the ranges given you are getting a bunch of stun added.

If I'm reading correctly I think this is "fine", in that the stun duration ramps up so that you're getting 3 seconds of stun for every 4 seconds (adjusted for mult) at the top end, but these aren't "the same".

Reply:
Apparently `mult` is `2` in a lag-free situation, meaning that (with your latest changes) you'll be permastunned (i.e. stun being added that matches the amount of time ticking by), with maybe very brief windows between ticks depending on variable lag, once you hit `mult * 2 SECONDS`.

I would look to confirm in your local build.

Reply:
Oh, i see. I changed as requested to the SECONDS defines.